app/cdap/components/RecipeManagement/ApplyRecipeView.tsx

```diff
@@ -0,0 +1,219 @@
+          sortBy={SortBy.UPDATED}
+          sortOrder={SortOrder.DESCENDING}
+          pageSize={6}
+          showPagination={true}
+          enableSorting={true}
+        />
+      </>
+    );
+  };
+
+  /* TO DO : Remove this function once Recipe Details UI component is integrated  */
+  const renderRecipeDetailsView = () => {
+    const Container = styled.div`
+      display: flex;
+    `;
+    const MainContainer = styled.div`
+      line-height: 2.5;
+    `;
+    const StepsTable = styled.table`
+      box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.2), 0px 2px 2px rgba(0, 0, 0, 0.14),
+        0px 1px 5px rgba(0, 0, 0, 0.12);
+      width: 100%;
+    `;
+    const Divider = styled.div`
+      width: 1px;
+      height: 20px;
+      background-color: #dadce0;
+      margin: 7px 15px;
+    `;
+    return (
+      <>
+        <MainContainer>
+          <div>
+            <strong>{selectedRecipe.recipeName}</strong>
+          </div>
+          <Container>
+            <span>
+              {`${selectedRecipe.recipeStepsCount} `} {T.translate(`${PREFIX}.recipeSteps`)}
+            </span>
+            <Divider></Divider>
+            <span>{format(selectedRecipe.updatedTimeMillis, TYPES.TIMESTAMP_MILLIS)}</span>
+          </Container>
+          <div>{selectedRecipe.description}</div>
+          <div className="table">
+            <StepsTable>
+              <tbody className="table table-hover">
+                <tr>
+                  <th>{'#'}</th>
+                  <th>{T.translate(`${PREFIX}.recipeSteps`)}</th>
+                </tr>
+                {Object.keys(selectedRecipe.directives).map((key, index) => (
+                  <tr className="table-row" key={`tablevalue-${index}`}>
+                    <td>{index}</td>
+                    <td>{selectedRecipe.directives[key] + ' ' + key}</td>
+                  </tr>
+                ))}
+              </tbody>
+            </StepsTable>
+          </div>
+        </MainContainer>
+      </>
+    );
+  };
+
+  const renderRecipeDetailsWithApply = () => {
+    const ButtonsContainer = styled.div`
+      display: flex;
+      justify-content: space-between;
+      max-width: 150px;
+    `;
+    return (
+      <Box>
+        {/* TO DO : Replace renderRecipeDetailsView()  with Recipe Details UI component once ready  */}
+        {renderRecipeDetailsView()}
+        <ButtonsContainer>
+          <PrimaryContainedButton
+            onClick={() => applySelectedRecipe(selectedRecipe.recipeId.recipeId)}
+            data-testid="recipe-apply-btn"
+          >
+            {T.translate(`${PREFIX}.applyRecipe`)}
+          </PrimaryContainedButton>
+          <PrimaryContainedButton
+            color="default"
+            onClick={() => setSelectedRecipe(null)}
+            data-testid="recipe-apply-cancel-btn"
+          >
+            {T.translate(`${PREFIX}.cancel`)}
+          </PrimaryContainedButton>
+        </ButtonsContainer>
+      </Box>
+    );
+  };
+  return (
+    <>
+      {
```

Comment:
why the curly braces around <Snackbar .../>?

Reply:
Missed while refactoring the code. removed it now.